src/DiIiS-NA/D3-GameServer/GSSystem/AISystem/Brains/AggressiveNPCBrain.cs

```diff
@@ -70,7 +71,7 @@ public override void Think(int tickCounter)
 					{
 						_target = monsters[0];
 						int powerToUse = PickPowerToUse();
-						if (powerToUse > 0)
+						if (powerToUse > 0) // FIXME: probably >= 0 as 0 can be a valid power?
```

Comment:
mistake? this method returns -1 if it didn't find an implemented power. So 0 probably is valid value